generators/skill/templates/function/node8/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "Update this description -- <%= projectName %> <%= functionName %>",
   "license": "UNLICENSED",
-  "main": "src/index.js",
+  "main": "./index.js",
```

Comment:
Not sure why this wasn't working correctly when being deployed.. might be overkill but this did work when Marcela was testing it.